src/main/java/uk/co/evoco/webdriver/utils/SelectBoxUtils.java

```diff
@@ -45,42 +49,46 @@ public static void itemByVisibleText(WebElement selectBox, String visibleText) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes